### DISCUSSION TITLE
automations

### DISCUSSION THREAD

**Discussion by marco-santulli** [Budibase/budibase#3744] [Help]
Hi,

sorry if this is a silly question, but I can't find a log of automation executions, is it somewhere to be found?

M

**Comment by mjashanks** [comment]
Hi Marco. We don't record automation runs at the moment. Definitely something that we need to add.

**Reply by marco-santulli** [reply]
Hi mike,

any chance you are going to add this? It is really difficult to get automations working and ensuring they don't have errors otherwise?

M
